k8s/activemq/chart/activemq/values.yaml

```diff
@@ -6,4 +6,6 @@ persistence:
   storageClass: null
   size: 5Gi
 consolePassword: null
-enableBashDebug: false
\ No newline at end of file
+enableBashDebug: false
+admin:
+  bindAllHosts: true
```

Comment:
For security purposes should the default be false?

Reply:
Default set as `false` .

`apptest/` schema.yaml has `true` as default only for testing purposes.